extensions/sensors/GetEnvironmentalSensors.h

```diff
@@ -75,7 +75,7 @@ class GetEnvironmentalSensors : public SensorBase {
   static std::shared_ptr<utils::IdGenerator> id_generator_;
 };
 
-REGISTER_RESOURCE(GetEnvironmentalSensors, "Provides sensor information from known sensors to include humidity and pressure data")
+REGISTER_RESOURCE(GetEnvironmentalSensors, "Provides sensor information from known sensors to include humidity and pressure data");
```

Comment:
Again, why did this work, if the semicolon is required?

Reply:
See the answer above. SMART_ENUM has the semicolon as part of the macro, and it is used consistently.